tskv/src/compaction/picker.rs

```diff
@@ -168,3 +358,138 @@ impl LevelCompatContext {
         Some((level, inputs))
     }
 }
+
+mod test {
+    use std::sync::Arc;
+
+    use crate::{
+        kv_option::TseriesFamOpt,
+        memcache::MemCache,
+        tseries_family::{ColumnFile, LevelInfo, TseriesFamily, Version},
+        TimeRange,
+    };
+
+    fn create_options(tsm_dir: &str) -> Arc<TseriesFamOpt> {
+        Arc::new(TseriesFamOpt {
+            base_file_size: 16 * 1024 * 1024,         // 16 MiB: 16777216
+            max_compact_size: 2 * 1024 * 1024 * 1024, // 2 GiB: 2147483648
+            tsm_dir: tsm_dir.to_string(),
+            max_level: 4,
+            level_ratio: 16_f64,
```

Comment:
remove unuse familyopt param

Reply:
Seems, some are for LSM compaction,  some are for TimeWindow compaction.